consumer-features/example-provider-hello-world/v3/example-provider-hello-world-js/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "example-hello-world-js",
+  "version": "0.1.0",
+  "dependencies": {
+    "axios": "^0.27.2",
+    "cors": "^2.8.5",
+    "express": "^4.18.1"
+  },
+  "scripts": {
+    "test": "mocha --exit --timeout 30000 src/provider.pact.spec.js"
+  },
+  "devDependencies": {
+    "@pact-foundation/pact": "^10.0.1",
+    "chai": "^4.3.6",
```

Comment:
Hmm, how would be good to expose this I wonder?
it might be nice to have a high level tldr: python tests using pytest, js using mocha and so on, just highlighting high level framework usage?